features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/backend-service-invoker.js

```diff
@@ -169,7 +171,7 @@ var backendServiceInvoker = function () {
         var wsRequest = new ws.WSRequest();
         var options = new Array();
         if (IS_OAUTH_ENABLED) {
-            var accessToken = session.get(constants.ACCESS_TOKEN_PAIR_IDENTIFIER).accessToken;
+            var accessToken = getAccessToken();
```

Comment:
What is the logic here? Having if (accessToken) is enough here as the double negation operators cancels each other.